calcom/cal.com
# chore: Clean Up Delete Credential Selected Calendar Error Message

## What does this PR do?



This PR aims to clean up an error message when deleting a calendar credential. Since last year, selected calendars have been deleted on cascade when the credential is deleted. There are still a lot of records that don't have this yet, so we have to keep these lines of code.

This is the error message that will be cleaned up.
![CleanShot 2023-11-13 at 13 47 34](https://github.com/calcom/cal.com/assets/65426560/135e9e5a-e368-4421-a3c9-bee8d6e90efa)


Fixes # (issue)



## Requirement/Documentation



- If there is a requirement document, please, share it here.
- If there is ab UI/UX design document, please, share it here.

## Type of change





## How should this be tested?



- Install two calendars of the same app (GCal)
- Select some calendars to check for conflicts for both credentials
- Delete one of the calendar apps
    - No error message should appear

## Mandatory Tasks


## Checklist



- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes
